plugins/mimecast/komand_mimecast/tasks/monitor_siem_logs/task.py

```diff
@@ -114,6 +136,24 @@ def _filter_and_sort_recent_events(
             ),
             key=itemgetter(EventLogs.FILTER_DATETIME),
         )
+
+        if len(task_output) >= MAX_EVENTS_PER_RUN:
+            start_point = last_log_line
+            end_point = start_point + MAX_EVENTS_PER_RUN
+
+            if end_point > len(task_output) - 1:
+                end_point = len(task_output) - 1
+                last_log_line = 0
+            else:
+                last_log_line = end_point
+
+            self.logger.info(
```

Comment:
should we log the filter time in here to see it's being maintained between runs?

Reply:
I have added this to the logger